index.Rmd

```diff
@@ -1,105 +1,228 @@
+I loaded required packages
 
-You can do bullets like this:
+```{r, message=F, warning=F}
+library(raster)
+library(tidyverse)
+library(sf)
+library(rpart)
+library(rpart.plot)
+library(rasterVis)
+library(mapedit)
+library(mapview)
+library(caret)
+library(forcats)
+library(sp)
+library(raster)
+```
 
-* The first most important thing
-* The second most important thing
-* The third most important thing
+I prepare the data by reading in the different bands that comprise the satelitte imagery. Each band refers to different spectrum and all bands below are from Lyot crater"V00823007" image.
 
-You can do numbers like this:
 
-1. The first most important thing
-2. The second most important thing
-3. The third most important thing
+```{r, message=F, warning=F}
 
-See [http://rmarkdown.rstudio.com/](http://rmarkdown.rstudio.com/) for all the amazing things you can do.
+band1 <- ("b01.tiff")
+band2 <- ("b02.tiff")
+band3 <- ("b03.tiff")
+band4 <- ("b04.tiff")
+band5 <- ("b05.tiff")
+band6 <- ("b06.tiff")
+band7 <- ("b07.tiff")
+band8 <- ("b08.tiff")
+band9 <- ("b09.tiff")
+```
 
 
-Here's my first code chunk.
+
+```{r, message=F, warning=F}
+library(raster)
+
+band1 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b01.tiff")
+band2 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b02.tiff")
+band3 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b03.tiff")
+band4 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b04.tiff")
+band5 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b05.tiff")
+band6 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b06.tiff")
+band7 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b07.tiff")
+band8 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b08.tiff")
+band9 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b09.tiff")
```

Comment:
Why do you read in the bands more than once? Is there a difference between these two sets of images?

Reply:
I understand remote sensing and the difference between bands.  You read in band1 as 
`band1 <- ("b01.tiff")`
and then overwrite it with 
`band1 <- raster("C:/Users/Owner/Desktop/sare  backup/c/user hadar/Desktop/GEO511/Tasks/geo511-tasks-hsare/geo511-2020-final-project/geo511-2020-project-hsare/selection_of_landingMining_sites/b01.tiff")`

There is no reason to read them in twice (the second one overwrites the first).